src/runtime_src/core/common/api/xrt_ip.cpp

```diff
@@ -223,11 +227,19 @@ class ip_impl
     return count++;
   }
 
+  //Address of ISR register in CU control register space
+  //This is based on contract between HLS/VITIS and XRT
```

Comment:
```suggestion
  // Address of ISR register in CU control register space
  // This is based on contract between HLS/VITIS and XRT
```
spacing everywhere.
On the meaning of the comment itself, I need to figure out what I could do with such a feature... :-)